### PR TITLE
Refactored Hemodynamics

### LINKED ISSUE #3429: Hemodynamic Parameters: Enhance content
- [ ] Club Pulse(bmp), Rhythm and description fields together in one section. It must be clear to the user that rhythm indicates heartbeat rhythm and the description is to note the description of such rhythm.
- [ ] Club together BP systolic and diastolic in one section called BP.
- [ ] Add SPO2 to this section. SPO2 is currently in the Ventilator parameters Section (https://care.coronasafe.in/facility/c92cc4df-0fea-4d37-9c24-1c9f07bfac23/patient/05d1be0f-51e6-4eab-8682-bc5515f5c863/consultation/aea6e0ab-7405-4186-b76e-b4e1aa3bbdca/daily_rounds/779a961f-9045-4a1f-b996-d4af459d7c59/update). DO NOT remove it from ventilator parameter. The same field must be mirrored in Vitals also.

### PR DESCRIPTION
**Pull request by skks1212**
Closes #3429

### CODE REVIEW

**Comment by nihal467** [comment]
@skks1212 merge the latest development

**Comment by shivankacker** [comment]
@nihal467 done 👍

**Comment by nihal467** [comment]
test approved

File: src/Components/Common/utils/ValidationUtils.res [hunk 1]
```diff
@@ -10,10 +10,13 @@ let isInputInRangeFloat = (minString, maxString, val) => {
...
 let isInputInRangeInt = (min, max, val) => {
-  let value = Js.Option.getWithDefault(min, val)
-  if value < min || value > max {
-    Some("Input outside range")
-  } else {
-    None
+  switch val {
+  | Some(value) =>
+    switch (value < min, value > max) {
+    | (true, _) => Some("Input less than " ++ string_of_int(1))
```
Comment (khavinshankar) [inline src/Components/Common/utils/ValidationUtils.res]: @skks1212 
should that 1 be hardcoded or was it suposed to be min?

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**